guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md

```diff
@@ -71,7 +71,7 @@ When your application is not in the production mode, you can set Magento to comp
 
 To set the compilation mode, do the following:
 <ol>
-<li>In the Magento Admin, navigate to <b>Stores</b> > <b>Configuration</b> > ADVANCED > <b>Developer</b>.</li>
+<li>In the Magento Admin, navigate to <b>Stores > Configuration > ADVANCED > Developer</b>.</li>
```

Comment:
We now use markdown formatting for lists. Please replace the HTML with the markdown version of lists: https://daringfireball.net/projects/markdown/syntax#list

Reply:
We use markdown for text formatting. Please replace `<b>` with `**` surrounding what you want to be bold.

And ensure that you do NOT include the `>` in the bolded text.

Reply:
You only have to make the change to the `css-preprocess.md` file in v2.1, as 2.2 and 2.3 versions are symlinked to 2.1. Please remove v2.2 and v2.3 changes.